README.md

```diff
@@ -147,6 +147,7 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [JOBBOX.io](https://landing.jobs/offers) – Filter -> Remote only.
   1. [Jobspresso](https://jobspresso.co/) * High-quality remote positions that are open and legitimate *
   1. [Landing.jobs](https://landing.jobs/offers) filter -> Remote only
+  1. [LaraJobs](https://larajobs.com) – Filter -> “*remote*”

```

Comment:
Maybe replace with https://larajobs.com/?location=&remote=1, instead?

Reply:
Sure, done that. Nice suggestion.